addons/web/static/src/views/view_components/view_scale_selector.js

```diff
@@ -13,7 +13,9 @@ export class ViewScaleSelector extends Component {
     static props = {
         scales: { type: Object },
         currentScale: { type: String },
+        showWeekends: { type: Boolean, optional: true },
         setScale: { type: Function },
+        toggleShowWeekends: { type: Function, optional: true },
```

Comment:
I know it's still a draft, but can you rename to `isWeekendVisible` and `toggleWeekendVisibility`?
I'm not a fan of the name of the toggle props.

Singular or plurial form, I don't really know what's the best🤔